test/distributed/optim/test_zero_redundancy_optimizer.py

```diff
@@ -1385,71 +1355,6 @@ def _test_ddp_zero_overlap(
                 # Ensure that this test runs independently
                 dist.barrier()
 
-    # NOTE: The test is skipped if using Windows since functional optimizers
-    # are not currently supported.
-    @common_distributed.skip_if_win32()
-    @common_distributed.requires_nccl()
-    @common_distributed.skip_if_no_gpu
-    @common_distributed.skip_if_rocm
-    @parametrize(
-        "use_gpu",
-        [True],
-        # Add `False` once the Gloo sync issue causing hangs is fixed
-        # See: https://github.com/pytorch/pytorch/issues/62300
-    )
-    @parametrize(
-        "use_interleaved_hook",
-        [False, True],
-    )
-    @parametrize(
-        "gradient_as_bucket_view",
-        [False, True],
-    )
-    @parametrize(
-        "static_graph",
-        [False, True],
-    )
-    @parametrize(
-        "shard_buckets",
-        [False, True],
-    )
-    def test_ddp_zero_overlap(
-        self,
-        use_gpu: bool,
-        use_interleaved_hook: bool,
-        gradient_as_bucket_view: bool,
-        static_graph: bool,
-        shard_buckets: bool,
-    ):
-        """
-        Check that overlapping DDP with ZeRO using the given method determined
-        by ``hook_constructor`` and ``shard_buckets`` and using the given ZeRO
-        and DDP arguments achieves parity with DDP using a local optimizer.
-        """
-        device = torch.device(self.rank) if use_gpu else torch.device("cpu")
-        backend = _get_backend_for_tests()
-        self.dist_init(self.rank, self.world_size, backend)
-        hook_constructor = (
-            hook_with_zero_step
-            if not use_interleaved_hook
-            else hook_with_zero_step_interleaved
-        )
-
-        # Disable DDP + ReplicatedTensor since ZeroRedundancyOptimizer
```

Comment:
This test looks like it's just disabling replicated tensor and then running the logic, so shouldn't we just remove the disable replicated tensor part versus remove the entire test?